datahelper.py

```diff
@@ -0,0 +1,19 @@
+import logging
+from google_drive_downloader import GoogleDriveDownloader as gdd
```

Comment:
I know this isn't in scope yet, but just remember to make a `requirements.txt` or something with all the dependencies for when this becomes a pip-package :slightly_smiling_face: